.github/workflows/release-nuget.yml

```diff
@@ -0,0 +1,28 @@
+name: Build and deploy NuGet package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.101
+    - name: Install dependencies
+      run: dotnet restore
+      
+    - name: Build
+      run: |
+        $test = "${{github.ref}}".trim().substring(11)
```

Comment:
It took me a minute to understand what this line was doing; maybe better for readability to use `'refs/tags/v'.Length` rather than `11`?

Alternately, maybe a good idea to add a format verification, to avoid publishing a bizarre version in the event of an issue with release drafter or someone mistakenly editing the tag name in a draft release:

```powershell
if ("${{github.ref}}".trim() -notmatch '^refs/tags/v(\d+\.\d+\.\d+)$') {
  exit 1
}
$version = matches[1]
dotnet build --no-restore --configuration Release -p:Version=$version
```

Reply:
That does look like way less of a hack. 
Will steal. :)